src/test/java/com/google/api/codegen/testdata/discoveries/csharp/csharp_translate.v2.json.baseline

```diff
@@ -26,18 +23,13 @@ namespace TranslateSample
         {
             TranslateService translateService = new TranslateService(new BaseClientService.Initializer
             {
-                ApplicationName = "Google-TranslateSample/0.1",
+                ApiKey = "",  // Update placeholder with desired API key.
 
-                // TODO: Change placeholder below to desired API key:
-                ApiKey = "{MY-API-KEY}",
+                ApplicationName = "Google-TranslateSample/0.1",
             });
 
-
-            // TODO: Change placeholder below to desired parameter value for the `List` method:
-
             // The text to detect
-            List<string> q = new List<string>();
-
+            List<string> q = new List<string>();  // TODO: Update placeholder value.
```

Comment:
Should we use an initializer here (and in Java) that's more readily extensible (to add values)?